crypto.go

```diff
@@ -132,9 +163,9 @@ func GenerateKeyPair() (EncryptionKeys, error) {
 
 // EncryptData encrypts a collection of data of string key and values using
 // Tozny v1 Record encryption, returning the encrypted data.
-func EncryptData(Data map[string]string, ak SymmetricKey) *map[string]string {
+func EncryptData(data map[string]string, ak SymmetricKey) *map[string]string {
```

Comment:
can the return type be *DecryptedData ?

Reply:
We probably want it to be EncryptedData yeah?